test/Groups/GrpAb.jl

```diff
@@ -11,3 +11,15 @@
   @test issurjective(g)
   @test order(codomain(g)) == 2
 end
+
+@testset "describe for GrpAbFinGen" begin
+  @test describe(abelian_group(GrpAbFinGen, Int[])) == "0"
+  @test describe(abelian_group(GrpAbFinGen, Int[0])) == "Z"
+  @test describe(abelian_group(GrpAbFinGen, Int[0, 0])) == "Z^2"
+  @test describe(abelian_group(GrpAbFinGen, Int[2])) == "Z/2"
+  @test describe(abelian_group(GrpAbFinGen, Int[0, 2])) == "Z/2 + Z"
+  @test describe(abelian_group(GrpAbFinGen, Int[0, 0, 2])) == "Z/2 + Z^2"
+  @test describe(abelian_group(GrpAbFinGen, Int[2, 4])) == "Z/2 + Z/4"
+  @test describe(abelian_group(GrpAbFinGen, Int[0, 2, 4])) == "Z/2 + Z/4 + Z"
+  @test describe(abelian_group(GrpAbFinGen, Int[0, 0, 2, 4])) == "Z/2 + Z/4 + Z^2"
```

Comment:
@micjoswig does that satisfy what you wanted for `GrpAbFinGen`?

Reply:
> @micjoswig does that satisfy what you wanted for `GrpAbFinGen`?

Yip, *much* better.